### PR TITLE
Create edematousAnatomicalEntity.yaml

### PR DESCRIPTION
**Pull request by srobb1**
pattern for entities with fluid swelling.

### CODE REVIEW

**Comment by matentzn** [comment]
I asked @nicolevasilevsky and @LCCarmody for a review of this pattern; sounds.. medical. Thanks @srobb1

**Comment by LCCarmody** [comment]
The structure/content is good for me.  I think the EQ works (but that isn't my strong suit).

May consider adding other synonyms like '%s edema'.

Also, for some terms, at least in the HP, we would want to override the term label for terms like 'lymphedema' and 'generalized edema'.  

Also, consider adding a synonym column to the vars list or data vars list.  (This last point may be true for multiple patterns.)

File: src/patterns/dosdp-dev/edematousAnatomicalEntity.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+classes:
+  abnormal: PATO:0000460
+  anatomical entity: UBERON:0001062
+  edematous: PATO:0001450
+
+relations:
+  inheres_in: RO:0000052
+  has_modifier: RO:0002573
+  has_part: BFO:0000051
+  part_of: BFO:0000050
```
Comment (nicolevasilevsky) [inline src/patterns/dosdp-dev/edematousAnatomicalEntity.yaml]: I don't think you need the part_of here, do you?

File: src/patterns/dosdp-dev/edematousAnatomicalEntity.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+annotations:
+  - annotationProperty: exact_synonym
+    text: "%s that is swollen with an excessive accumulation of fluid"
+    vars:
+     - anatomical_entity
+  - annotationProperty: exact_synonym
+    text: "%s that exhibits an excessive accumulation of extracellular fluid"
+    vars:
+     - anatomical_entity
```
Comment (nicolevasilevsky) [inline src/patterns/dosdp-dev/edematousAnatomicalEntity.yaml]: you could also add a synonym with the UK spelling: 'oedema in %s'

File: src/patterns/dosdp-dev/edematousAnatomicalEntity.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+name:
+  text: "edema in %s"
+  vars:
+   - anatomical_entity
+
+def:
+  text: "An abnormal %s with edema."
+  vars:
+    - anatomical_entity
```
Comment (nicolevasilevsky) [inline src/patterns/dosdp-dev/edematousAnatomicalEntity.yaml]: what about this for a text def:
An abnormal accumulation of fluid in the %s.

(I am just modeling this after HPO)

File: src/patterns/dosdp-dev/edematousAnatomicalEntity.yaml (hunk 2 above)
Comment (nicolevasilevsky) [reply src/patterns/dosdp-dev/edematousAnatomicalEntity.yaml]: also, what about the synonym: '%s edema'

File: src/patterns/dosdp-dev/edematousAnatomicalEntity.yaml (hunk 3 above)
Comment (matentzn) [reply src/patterns/dosdp-dev/edematousAnatomicalEntity.yaml]: I would also like to see this in the definition of the pattern! I mean the phrase "accumulation of fluid".

Comment (srobb1) [reply src/patterns/dosdp-dev/edematousAnatomicalEntity.yaml]: Thanks for fixing this up!! Looks GOOD!!! Honestly, I didn't know what edema meant until I saw it in my phenotype list and I looked it up.

**Pull request merged**